back/CinemaReservation.BusinessLayer/Contracts/ISecurityService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CinemaReservation.BusinessLayer.Contracts
+{
+    public interface ISecurityService
+    {
+        byte[] GetHash(string password, byte[] salt);
+        (byte[] passwordHash, byte[] salt) GetPasswordHashAndSalt(string password);
```

Comment:
I do not recomment to use tuples for public APIs. Custom classes or interfaces should be used instead